### ISSUE TITLE
be smart about number of layers in models

### ISSUE THREAD

**Issue by wdwatkins** [USGS-R/pgml_temperature_prediction#16]
Code is already there in `R/get_base_lake_nml.R`

**Comment by jread-usgs** [comment]
Does this issue need more context? It might be written in a way that only makes sense to you.

**Comment by wdwatkins** [comment]
referring to this:

**Comment by jread-usgs** [comment]
yes, good idea. We hadn't run into this before because we always had *way too many* layers.

**Comment by wdwatkins** [comment]
200 seems to be a safe number so far.

**Comment by wdwatkins** [comment]
I suppose this should theoretically work now that the precipitation units are correct, although Luke pointed out we could just convert the netcdf files to netcdf4 so they use compression — this probably saves more space than messing with layers.  Although I suppose if wanted to maximize disk space efficiency we could use both.  

`nccopy -knc7 -d9 output.nc output_4.nc`  adds max compression through the `-d` option.  Converted a 1.7GB file to 48M.  Might make reading a bit slower, but doesn't seem particularly noticeable.

**Comment by jread-usgs** [comment]
nice! That is a huge gainer

**Comment by wdwatkins** [comment]
After fixing the precip data units, there were still a few lakes that failed (shallow ones again).  I was able to get them to run by increasing the layer threshold to `ceiling(max_depth/min_thick * 1.5)`.

**Comment by jread-usgs** [comment]
We might want to propagate that change back into the functions that create the base nmls for each lake.
